_posts/2016-08-19-CD-for-UX-and-PO.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+author: HannuToi
+title: Continuous delivery for Product Owners and UX designers
+excerpt: Continuous Delivery is powerful tool for UX designers and Product Owners. 
+tags:
+- user experience
+- product owner
+- continuous delivery
+---
+
+It’s fun to create product and service concepts. You can use your imagination and come up with great ideas for products and their features. You can even go all out and try to validate those ideas with real users using html and paper prototypes,. Emphasizing “try”.  Even at the best possible scenario you will be guesstimating based on your expertise, user’s feelings or your clients opinions. And based on this guesstimation you will be making some of the biggest decisions on your project.
+
+## SCRUMing it
+
+In traditional waterfall and SCRUM projects you’re now in bit of a pickle. Your team will start working on product or a feature. And that feature is specified based on assumptions. It could be a year from now before that feature would be actual used. Or in worst case – never. You might be building completely irrelevant feature based on client’s or user’s idea in some specifications workshop that took place some Friday afternoon half a year ago. 
+
+“But doesn’t Agile solve this problem?” Not really. By simply doing your project in agile mode doesn’t give you the real insight in what should be done. You will have more flexibility with client’s demands, which however, won’t be based on actual information unless you are able to collect feedback from real usage. And even in that case, by using SCRUM, you can propably react to the feedback at earliest in the next sprint – which could take a month (or so).
```

Comment:
insight in what should -> insight into  tai insight to
actual information unless -> actual information, unless
propably react -> probably react